presto-native-execution/presto_cpp/main/PrestoServer.h

```diff
@@ -87,9 +87,9 @@ class PrestoServer {
  protected:
   /// Hook for derived PrestoServer implementations to add/stop additional
   /// periodic tasks.
-  virtual void addAdditionalPeriodicTasks(){};
+  virtual void addAdditionalPeriodicTasks(){}
```

Comment:
@tdcmeehan  @majetideepak This is a good change. Should we make it (in a separate PR)?